share/ipld/get_shares.go

```diff
@@ -44,12 +44,13 @@ func GetShares(ctx context.Context, bg blockservice.BlockGetter, root cid.Cid, s
 func GetSharesByNamespace(
 	ctx context.Context,
 	bGetter blockservice.BlockGetter,
-	root cid.Cid,
+	root []byte,
```

Comment:
Don't get why is that needed to be part of this PR

Reply:
because ipld.FilterRootByNamespace is gone and no longer returns cids. We use indexes and row roots instead now

Reply:
Yeah, but you could calll MustRootToCID before calling GetShareByNamespace

Reply:
ipld.GetSharesByNamespace is used in File pkg to build proofs from the tree. I don't think File needs to be aware of cids, so construction of cid moved into ipld pkg

Reply:
kk